tests/integration_tests/test_observation_times.py

```diff
@@ -0,0 +1,159 @@
+"""
+Tests behavior of matching response times to observation times
+"""
+from contextlib import redirect_stderr
+from datetime import datetime
+from io import StringIO
+from textwrap import dedent
+
+import hypothesis.strategies as st
+import numpy as np
+import pytest
+from hypothesis import assume, given, settings
+
+from ert.cli import ES_MDA_MODE
+from ert.cli.main import ErtCliError
+from tests.unit_tests.config.observations_generator import summary_observations
+from tests.unit_tests.config.summary_generator import summaries
+
+from .run_cli import run_cli
+
+start = datetime(1969, 1, 1)
+observation_time = datetime(2024, 1, 1)
+epsilon = 0.1
+
+
+@settings(max_examples=3)
+@given(
+    responses=st.lists(
+        summaries(
+            start_date=st.just(start),
+            time_deltas=st.just(
+                [((observation_time - start).total_seconds() + epsilon) / 3600]
+            ),
+            summary_keys=st.just(["FOPR"]),
+            use_days=st.just(False),
+        ),
+        min_size=2,
+        max_size=2,
+    ),
+    observation=summary_observations(
+        summary_keys=st.just("FOPR"),
+        std_cutoff=10.0,
+        names=st.just("FOPR_OBSERVATION"),
+        dates=st.just(observation_time),
+        time_types=st.just("date"),
+    ),
+    std_cutoff=st.floats(min_value=0.0, max_value=1.0),
+    enkf_alpha=st.floats(min_value=3.0, max_value=10.0),
+)
+def test_small_time_mismatches_are_ignored(
+    responses, observation, tmp_path_factory, std_cutoff, enkf_alpha
+):
+    tmp_path = tmp_path_factory.mktemp("summary")
+    (tmp_path / "config.ert").write_text(
+        dedent(
+            f"""
+            NUM_REALIZATIONS 2
+            ECLBASE CASE
+            SUMMARY FOPR
+            MAX_SUBMIT 1
```

Comment:
Why 1?

Reply:
Just used so that we don't retry (saves time).